networkdb/cluster.go

```diff
@@ -141,6 +142,7 @@ func (nDB *NetworkDB) clusterInit() error {
 		{config.GossipInterval, nDB.gossip},
 		{config.PushPullInterval, nDB.bulkSyncTables},
 		{retryInterval, nDB.reconnectNode},
+		{deadNodeReapInterval, nDB.reapDeadNode},
```

Comment:
I think we should run this go-routine for a lesser period (say every hour) and delete failed nodes which have been in the `failedNodes` for 24 hours by adding a setting a failed timestamp on the node itself.  The current logic is will be too imprecise.